BasicXrApp.sln

```diff
@@ -5,14 +5,15 @@ VisualStudioVersion = 15.0.28307.421
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "BasicXrApp_uwp", "samples\BasicXrApp\BasicXrApp_uwp.vcxproj", "{1B09B21C-2D7A-4278-81C8-84A47D5834A7}"
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "BasicXrApp_win32", "samples\BasicXrApp\BasicXrApp_win32.vcxproj", "{A75A907B-8952-4ED2-BC2D-A68F09CEBD83}"
+EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "xrUtility", "xrUtility", "{D71728CE-842F-4FA7-94AE-01AEA60BC45A}"
 	ProjectSection(SolutionItems) = preProject
-		samples\XrUtility\XrError.h = samples\XrUtility\XrError.h
-		samples\XrUtility\XrExtensions.h = samples\XrUtility\XrExtensions.h
-		samples\XrUtility\XrHandle.h = samples\XrUtility\XrHandle.h
-		samples\XrUtility\XrMath.h = samples\XrUtility\XrMath.h
-		samples\XrUtility\XrString.h = samples\XrUtility\XrString.h
-		samples\XrUtility\XrToString.h = samples\XrUtility\XrToString.h
+		shared\XrUtility\XrError.h = shared\XrUtility\XrError.h
+		shared\XrUtility\XrExtensions.h = shared\XrUtility\XrExtensions.h
+		shared\XrUtility\XrHandle.h = shared\XrUtility\XrHandle.h
+		shared\XrUtility\XrMath.h = shared\XrUtility\XrMath.h
+		shared\XrUtility\XrString.h = shared\XrUtility\XrString.h
```

Comment:
was XrToString.h removed?

Reply:
The header is still here, it's not referenced by this project, therefore removed from this list.